src/plugins/intel_gpu/src/graph/layout_optimizer.cpp

```diff
@@ -1887,6 +1908,25 @@ void layout_optimizer::select_preferred_formats_for_onednn(program_node& node, d
                 }
             }
 
+            // In conv-permute pattern, sets the output format of conv to byxf so that permute can be optimized.
+            // ex) oneDNN convolution -> (byxf) -> permute -> (bfyx) -> output
+            //     output layout of convolution: byxf [b:1, f:128, y:2, x:2]
+            //     output layout of permute:     bfyx [b:1, f:2, y:2, x:128]
+            // In this case, it can be handled by changing only the shape of permute without the kernel execution.
+            if (node.get_output_layout().get_rank() == 4 && node.get_users().front()->is_type<permute>()) {
+                auto& pnode = node.get_users().front()->as<permute>();
+                auto can_optimize_permute = pnode.get_users().size() == 1 && pnode.get_dependencies().size() == 1
+                    && !pnode.is_output() && pnode.get_dependency(0).get_output_layout().is_static()
```

Comment:
I think we need to check whether `node.get_users().size() == 1` instead.

Reply:
Added with another condition for concat.